### ISSUE TITLE
Neural Networks, Manifolds, and Topology 

### ISSUE THREAD

**Issue by ShakirKhurshid** [erodola/DLAI-s2-2021#4]
found this interesting article on Topology, Hemomorphismss, Manifolds etc etc. you might find it interesting!
It gives a visual insight into what's happening around with the data in  the Neural Networks

[Link](https://colah.github.io/posts/2014-03-NN-Manifolds-Topology/)

**Comment by erodola** [comment]
I agree that this is a well written, useful article. Thanks for sharing!
